test/common/storagelib.py

```diff
@@ -45,22 +45,6 @@ def step():
 
         self.browser.wait(step)
 
-    def add_ram_disk(self, size=50):
```

Comment:
storagelib.py is *already* official test API, we use it in c-machines and anaconda-webui. Plus, this is a mix-in, so test classes can easily add it.

Reply:
Added :)